modules/ztp-image-based-upgrade-prep-oadp.adoc

```diff
@@ -16,6 +16,7 @@ Prepare your {oadp-short} resources to restore your application after an upgrade
 * Deploy a version of {lcao} that is compatible with the version used with the seed image.
 * Install the {oadp-short} Operator, the `DataProtectionApplication` CR, and its secret on the target cluster.
 * Create an S3-compatible storage solution and a ready-to-use bucket with proper credentials configured. For more information, see "Installing and configuring the {oadp-short} Operator with {ztp}".
+* The `openshift-adp` namespace for the OADP `ConfigMap` object must exist on all managed clusters and the hub for the OADP `ConfigMap` to be generated and copied to the clusters.
```

Comment:
```suggestion
* The `openshift-adp` namespace for the OADP `ConfigMap` object must exist on all managed clusters and the hub for the OADP `ConfigMap`, so that X can generate the namespace and copy it to the required clusters.
```